packages/cli/commands/process.js

```diff
@@ -0,0 +1,84 @@
+const path = require('path');
+const fs = require('fs');
+const { createIgnoreFilter } = require('@hubspot/cli-lib/ignoreRules');
+const { isAllowedExtension } = require('@hubspot/cli-lib//path');
+const { logger } = require('@hubspot/cli-lib/logger');
+const { walk } = require('@hubspot/cli-lib/lib/walk');
+const { getThemeJSONPath } = require('@hubspot/cli-lib/lib/files');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const {
+  FieldsJs,
+  isProcessableFieldsJs,
+} = require('@hubspot/cli-lib/lib/handleFieldsJs');
+const i18nKey = 'cli.commands.process';
+
+exports.command = 'process';
```

Comment:
I think we will want to pull in some design help to figure out where this command should live. We are moving towards creating buckets of commands since the scope of the CLI is increasing beyond the CMS. So there will be a `hs cms` bucket. I'm thinking that maybe we should put this in there.

cc/ @markhazlewood